services/QuillLMS/app/models/segment_integration/user.rb

```diff
@@ -73,6 +78,16 @@ def school_params
       end
     end
 
+    def onboarding_group_params
+      return {} if admin? || !teacher?
+
+      onboarding_slides_milestone = Milestone.find_by_name(Milestone::TYPES[:see_onboarding_slides_v1])
+
+      {
+        onboarding_group: user_milestones.find_by(milestone: onboarding_slides_milestone) ? SLIDES_V1 : DEFAULT
```

Comment:
`exists?` is more appropriate (and faster) here since the record isn't being used:

user_milestones.exists?(milestone: onboarding_slides_milestone)